classes/class-wc-connect-api-constants.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * A class for constants used by the API.
+ */
+class WC_Connect_API_Constants {
+	// These are used for WCS&T to WCShipping migration.
+	public const MIGRATION_STATE_NOT_STARTED = 1;
+	public const MIGRATION_STATE_STARTED = 2;
+	public const MIGRATION_STATE_COMPLETED = 3;
+	public const MIGRATION_STATE_FAILED = 4;
```

Comment:
WDYT about something like this?

```suggestion
class WC_Connect_WCST_To_WCShipping_Migration_State_Enum {

	public const NOT_STARTED = 1;
	public const STARTED = 2;
	public const COMPLETED = 3;
	public const FAILED = 4;
```

Reply:
Updated in 85a7e614